hsm/README.md

```diff
@@ -8,7 +8,7 @@ between the tiers (pools). Here is how to use the tool:
 1. Set the following environment variables:
 
    ```bash
-   export CLIENT_PROF_OPT="<0x7000000000000001:0x480>"   # profile id
+   export CLIENT_PROFILE="<0x7000000000000001:0x480>"    # profile id
    export CLIENT_HA_ADDR="172.18.1.24@o2ib:12345:34:101" # ha-agent address
    export CLIENT_LADDR="172.18.1.24@o2ib:12345:41:322"   # local address
    export CLIENT_PROC_FID="<0x7200000000000001:0xdf>"    # process id
```

Comment:
Do you like to change this to:
CLIENT_PROCESS
?
Or change "CLIENT_PROFILE" to "CLIENT_PROF_FID"?

Or it is OK to keep it.

Reply:
Let's leave it like this for now. I think PROC_FID is not as bad as PROF_OPT.